### PR TITLE
Pluribus Networks vrouter bgp module with unit test cases

### PR DESCRIPTION
**Pull request by rajaspachipulusu17**
##### SUMMARY

Pluribus Network vrouter bgp module with unit test cases

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

Point feature of vrouter bgp module
pn_vrouter_bgp.py with unit tests.
##### ADDITIONAL INFORMATION

```paste below
ansible 2.7.5
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.12 (default, Nov 12 2018, 14:36:49) [GCC 5.4.0 20160609]
```

### CODE REVIEW

**Comment by rajaspachipulusu17** [comment]
@justjais @ganeshrn @NilashishC  i didnt get this issue. Can some one point me what is the issue here?

Thanks,
Rajas

File: lib/ansible/modules/network/netvisor/pn_vrouter_bgp.py [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+        pn_route_map_in=dict(required=False, type='str'),
+        pn_soft_reconfig_inbound=dict(required=False, type='bool'),
+        pn_override_capability=dict(required=False, type='bool'),
+        pn_max_prefix_warn_only=dict(required=False, type='bool'),
+        pn_ebgp_multihop=dict(required=False, type='str'),
+        pn_remote_as=dict(required=False, type='str'),
+        pn_prefix_list_out=dict(required=False, type='str'),
+        pn_no_route_map_out=dict(required=False, type='str'),
+        pn_no_route_map_in=dict(required=False, type='str'),
+    ),
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_vrouter_bgp.py]:
```suggestion
    )
```

File: lib/ansible/modules/network/netvisor/pn_vrouter_bgp.py [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+    pn_remote_as: 65000
+    pn_bfd: false
+    pn_allowas_in: true
+"""
+
+RETURN = """
+command:
+  description: the CLI command run on the target node.
+  returned: always
+  type: string
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_vrouter_bgp.py]:
```suggestion
  type: str
```

Comment (rajaspachipulusu17) [reply lib/ansible/modules/network/netvisor/pn_vrouter_bgp.py]: done

**Comment by rajaspachipulusu17** [comment]
Can someone merge if there is no issue? @justjais @ganeshrn .

Thanks,
Rajas

**Comment by rajaspachipulusu17** [comment]
Can this be merged ?

**Comment by Akasurde** [comment]
@rajaspachipulusu17 Thanks for the contribution.

**Pull request merged**